keyboards/nasu/info.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Needs to be redone without rotation.
![image](https://user-images.githubusercontent.com/4781841/146100975-058a7d68-c3b4-41c0-bd47-d1afa201c322.png)

Reply:
Fixed